src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -171,6 +171,11 @@ public PriorityQueue(IEnumerable<(TElement Element, TPriority Priority)> items,
         /// </summary>
         public int Count => _size;
 
+        /// <summary>
+        ///  Gets the total numbers of elements the internal data structure can hold without resizing.
```

Comment:
It's called "backing storage" in EnsureCapacity:
https://github.com/dotnet/runtime/blob/34982753dcf92df95e3f70806e2848c6fed4885c/src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs#L588
It'd be good to use the same terminology here:
```suggestion
        ///  Gets the total numbers of elements the queue's backing storage can hold without resizing.
```

Reply:
https://github.com/search?q=repo%3Adotnet%2Fruntime%20%22the%20internal%20data%20structure%22&type=code

then all of these should be changed too